site/app/config/packages/old_sound_rabbit_mq.yml

```diff
@@ -30,3 +33,13 @@ old_sound_rabbit_mq:
             exchange_options: {name: 'update-project-info', type: fanout}
             queue_options:    {name: 'update-github-metadata'}
             callback:         update_github_metadata_consumer
+        web_notification:
+            connection:       default
+            exchange_options: {name: 'send-notification', type: fanout}
+            queue_options:    {name: 'send-notification'}
+            callback:         web_notification_consumer
+        email_notification:
+            connection:       default
+            exchange_options: {name: 'send-notification', type: fanout}
+            queue_options:    {name: 'send-notification'}
+            callback:         email_notification_consumer
```

Comment:
Isn't that callback the default and can be omitted?

Reply:
Just checked that it expects a `callback` to be there else an error is thrown